salt/metalk8s/salt/minion/installed.sls

```diff
@@ -4,6 +4,15 @@ include :
   - metalk8s.repo
 
 Install salt-minion:
+{%- if grains['os'] == 'CentOS' %}
```

Comment:
Not filter on `os` but on `osfamily` please, as we will, likely, support RedHat (maybe one day)

Reply:
Change done.